html/infrastructure/urls/resolving-urls/query-encoding/resources/resolve-url.js

```diff
@@ -1,6 +1,34 @@
 // NOTE: this file needs to be split up rather than expanded. See ../location.sub.html for some
 // extracted tests. Tracked by https://github.com/web-platform-tests/wpt/issues/4934.
 
+/*
+* help:
```

Comment:
@cvrebert @zcorpan have you been using the test-level help metadata for these tests in any way? Ideas about how it could be used? I'm not sure if it's worth preserving or not as machine-readable metadata.

Reply:
I've written tests with `help` but it's likely less useful than a js line comment with a spec link (the only use I imagine is for someone debugging a test).